plugins/pulp_rpm/plugins/importers/yum/report.py

```diff
@@ -1,23 +1,21 @@
-# -*- coding: utf-8 -*-
-
 from gettext import gettext as _
 import logging
 
 from pulp_rpm.common import constants
-from pulp_rpm.plugins.db import models
+from pulp_rpm.common import ids
 
 
 _logger = logging.getLogger(__name__)
 
 type_done_map = {
-    models.RPM.TYPE: 'rpm_done',
-    models.SRPM.TYPE: 'rpm_done',
-    models.DRPM.TYPE: 'drpm_done',
+    ids.TYPE_ID_RPM: 'rpm_done',
```

Comment:
+1 to using `models.<model_name>._content_type_id.default` instead of constants from `ids`. I've made this change for pre-merge.